### ISSUE TITLE
Request: can the file format be something "standard"?

### ISSUE THREAD

**Issue by philschatz** [openstax/manifestly#3]
- GitHub Flavored Markdown `openstax/tutor-js@abc123cc45`
- JSON `{"openstax/tutor-js": "abc123cc45"}`

I guess just removing the spaces would be enough (and backwards-compatible).

If there is interest, I can try to make a PR (and make sure the old format still works too)

**Comment by jpslav** [comment]
Yeah, I know, spaces are blah -- was just my fairly lame attempt to make it slightly more human readable (to inspect that the manifest contains what you expect).

But I could go without spaces too.  @cnuber any opinion?

Here are the [relevant lines](https://github.com/openstax/manifestly/blob/master/lib/manifestly/manifest_item.rb#L33-L38) in manifestly.

**Comment by philschatz** [comment]
lol, you strip the space anyway :smile:

**Comment by jpslav** [comment]
of course I do :-) like i said, for human reading only :-)

let's just leave it for now.
